docs/src/shared/localeProp.js

```diff
@@ -0,0 +1,9 @@
+const localeProp = {
+  name: "locale",
+  type: "string",
+  defaultValue: "undefined",
+  description:
+    "One of the supported locale strings: 'enUS' (US English), 'de' (German), 'fr' (French), 'es' (Spanish), 'ptBR' (Brazil Portuguese), 'da' (Dutch), 'pl' (Polish)"
```

Comment:
These could probably eventually go into a docs page on localization (?) But for now I guess its most convenient along side the prop so I've put it here.

Reply:
Yeah we'll do a docs page on localization soon. For now this is good.